backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/types/AcceptingStore.kt

```diff
@@ -15,6 +15,8 @@ data class AcceptingStore(
     val telephone: String?,
     val website: String?,
     val discount: String?,
+    val freinetId: Int?,
+    val districtName: String?,
```

Comment:
Not sure about naming here. We want two separate properties, right? And not just one `issuerId` containing either the freinet id or the district name?

Reply:
Merging those will be part of an other task